### PR TITLE
[css-backgrounds] Add test for background-clip on multi-line inline elements

### PR DESCRIPTION
**Pull request by litherum**
Currently, all browsers fail this test, because none implement both `background-clip: text` and `fill-color`.

If you replace those properties with `-webkit-background-clip` and `-webkit-text-fill-color`, all browsers (and WebKit after https://bugs.webkit.org/show_bug.cgi?id=230335) pass. But since those are both non-standard properties, I don't believe they have any place in WPT.

(I'm not sure why I can't add @LeaVerou as a reviewer to this PR.)

### CODE REVIEW

File: css/css-backgrounds/background-clip/clip-text-multi-line.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<head>
+<title>CSS Test: background-clip: text paints multi-line text in the wrong place</title>
+<link rel="help" href="https://drafts.csswg.org/css-backgrounds-4/#valdef-background-clip-text">
```
Comment (dbaron) [inline css/css-backgrounds/background-clip/clip-text-multi-line.html:5]: Maybe have a second link to https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-color ?

File: css/css-backgrounds/background-clip/clip-text-multi-line.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
```
Comment (dbaron) [inline css/css-backgrounds/background-clip/clip-text-multi-line.html:2]: I don't think you want reftest-wait here -- it also looks like nothing removes it.  Same for the reference.

File: css/css-backgrounds/background-clip/clip-text-multi-line-ref.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<head>
+<title>CSS Test: background-clip: text paints multi-line text in the wrong place</title>
+<link rel="help" href="https://drafts.csswg.org/css-backgrounds-4/#valdef-background-clip-text">
```
Comment (dbaron) [inline css/css-backgrounds/background-clip/clip-text-multi-line-ref.html:5]: I think generally we don't include a `<link rel=help>` in references, so probably better to remove this.  (Having it in the test is sufficient, and sometimes references are used for multiple tests that want different `<link rel=help>`.)

**Review by dbaron — APPROVED** [review]
Looks good to me with the changes mentioned.

**Comment by litherum** [comment]
https://bugs.webkit.org/show_bug.cgi?id=230335 indicates that there should be an additional test added to this PR.

**Comment by TalbotG** [comment]
Consider the following as a global suggestion.

At line 16, we have `font: 48px "Ahem";`
It should be noted that it is needed to always define the line-height
when declaring a shorthand font with Ahem font and best if it is
a multiple of 5px.

> because none implement (...) `fill-color`.

Then how about replacing
line 18: `fill-color: rgba(0, 0, 0, 0);`
with
line 18: `color: transparent`;

The current test is in my humble opinion unneedlessly complicated and not streamlined (3 selectors and 27 declarations). The pass condition is also weird when it states "is (roughly) half as wide" when it must be **half** as wide, not _roughly_.

Quick example of same test (1 selector and 7 declarations):
http://www.gtalbot.org/BrowserBugsSection/CSS3Backgrounds/clip-text-multi-line-GT.html

http://www.gtalbot.org/BrowserBugsSection/CSS3Backgrounds/reference/clip-text-multi-line-GT-ref.html

**Pull request merged**